app/MMB/src/scenes/timeline/events/LeaveForAirportTimelineEvent.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+
+import * as React from 'react';
+import idx from 'idx';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+import {
+  Translation,
+  DateFormatter,
+  TranslationFragment,
+} from '@kiwicom/mobile-localization';
+import { Text, Color } from '@kiwicom/mobile-shared';
+
+import type { LeaveForAirportTimelineEvent as LeaveForAirportTimelineEventType } from './__generated__/LeaveForAirportTimelineEvent.graphql';
+import TimelineEvent from '../TimelineEvent';
+import TimelineEventIcon from '../TimelineEventIcon';
+import DateLocation from '../TimelineEventDateLocation';
+
+type Props = {|
+  +data: LeaveForAirportTimelineEventType,
+|};
+
+const LeaveForAirportTimelineEvent = (props: Props) => {
```

Comment:
For the future reference: internals of this component looks also very similar to other events. Maybe you could reuse some parts?

Reply:
Yes, you're right. I will abstract this a bit more. Also, I realised that for now, I have timestamps for all TimelineEvents. So this should probably be dealt with in `TimelineEvent.js`. And that means I can remove some Relay Fragment containers as they are then useless. (I think in the current state, it is only justified to use it for LeaveForAirport, as `location` field is specific to this event). Thoughts?

Reply:
Yes, definitely. You don't need to work with the data from API here. You only need to tweak here styles, icons and translations and this moment.